frontend/components/collagephotos.js

```diff
@@ -0,0 +1,99 @@
+import { Image } from "react-grid-gallery";
+
+// images
+import Map_Page from "../images/featured/map page.png";
+import Walking_Man from "../images/featured/BHVP_PH_CetaitParis_DP_0122_01_078.jpg";
+import Car from "../images/featured/BHVP_PH_CetaitParis_DP_0122_01_008.jpg";
+import Staring_Man from "../images/featured/BHVP_PH_CetaitParis_DP_0122_01_024.jpg";
+import Roof from "../images/featured/BHVP_PH_CetaitParis_DP_0122_01_002.jpg";
+import Neighbourhood from "../images/featured/BHVP_PH_CetaitParis_DP_0122_01_012.jpg";
+import House from "../images/featured/BHVP_PH_CetaitParis_DP_0122_01_030.jpg";
+
+
+
+if (window.matchMedia('(min-width: 769px)').matches) {
+    // screens like laptops
+    var width = (window.screen.width*window.devicePixelRatio)/1;
+	var height = (window.screen.height*window.devicePixelRatio)/1.5;
+} else if (window.matchMedia('(min-width: 481px)').matches){
+    // screens like tablets
+    var width = (window.screen.width*window.devicePixelRatio)/3.5;
+	var height = (window.screen.height*window.devicePixelRatio)/1;
+	
+}else if (window.matchMedia('(min-width: 320px)').matches){
+    // phone screens
+    var width = (window.screen.width*window.devicePixelRatio)/4;
+	var height = (window.screen.height*window.devicePixelRatio)/9;
+}
+
+
+
+// const width = (window.screen.width*window.devicePixelRatio)/4;
+// const height = (window.screen.height*window.devicePixelRatio)/6;
+
+export const images = [
```

Comment:
It's good practice to reduce redundancy in your code. Instead of copying the `width` and `height` for each image, you can build this list of JSON objects programmatically.